api/Makefile

```diff
@@ -22,6 +22,14 @@ upgrade-db:
 	vagrant ssh -c "cd /vagrant/repos/snailx_api/api && . /vagrant/venv/bin/activate; \
 		python manage.py db upgrade"
 
+dummy-data-save-db:
+	vagrant ssh -c "pg_dump -h localhost -p 5432 -U dev
+		-d snailx -T alembic_version -f /vagrant/repos/snailx_api/api/db/dummy_data.dump --format custom"
+
+dummy-data-load-db:
```

Comment:
This command with init-db, migrate-db will populate the database, however, there are several errors may need looking into. Example below:
 ```pg_restore: creating SEQUENCE "public.round_id_seq"
pg_restore: [archiver (db)] Error from TOC entry 204; 1259 2237780 SEQUENCE round_id_seq fqtkemerxaamnr
pg_restore: [archiver (db)] could not execute query: ERROR:  syntax error at or near "AS"
LINE 2:     AS integer
            ^
    Command was: CREATE SEQUENCE public.round_id_seq
    AS integer
    START WITH 1
    INCREMENT BY 1
    NO MINVALUE
    NO MAXVALUE
    C...```